File: android/Dev/app/src/main/java/com/dxmnd/mos/dev/retrofit/service/RetrofitService.java
package com.dxmnd.mos.dev.retrofit.service;

import com.dxmnd.mos.dev.retrofit.model.ResponseModel;

import java.util.HashMap;
import java.util.List;

import io.reactivex.Observable;
import retrofit2.http.FieldMap;
import retrofit2.http.POST;

public interface RetrofitService {
    @POST("/")
    Observable<List<ResponseModel>> login(@FieldMap HashMap<String, String> map);
}

File: android/RightNow/app/src/main/java/dxmnd/com/rightnow/util/retrofit/RetrofitService.kt
package dxmnd.com.rightnow.util.retrofit

import dxmnd.com.rightnow.util.GET_STAIONS_BY_ROUTE_LIST
import retrofit2.http.GET
import retrofit2.http.Query
import retrofit2.http.QueryMap
import java.util.*

interface RetrofitService {
    @GET(GET_STAIONS_BY_ROUTE_LIST)
    fun getBusRoute(@QueryMap map : Map<String, String>) : Observable
}